developer/build.sh

```diff
@@ -40,4 +40,9 @@ builder_describe \
 
 builder_parse "$@"
 
+builder_describe_outputs \
+  configure  /developer/src/tike/xml/layoutbuilder/keymanweb-osk.ttf
+
+builder_run_action configure cp "$KEYMAN_ROOT/common/resources/fonts/keymanweb-osk.ttf" "$KEYMAN_ROOT/developer/src/tike/xml/layoutbuilder/"
```

Comment:
Sadly, developer/build.sh is not run in the Developer CI build -- this change needs to be in tike/Makefile.

Reply:
I'll give it a go